src/test/scala/uk/gov/hmrc/test/ui/pages/BasePage.scala

```diff
@@ -90,4 +92,14 @@ trait BasePage extends Matchers with BrowserDriver{
     driver.findElement(selector).clear()
     driver.findElement(selector).sendKeys(value)
   }
+
+  def clickForAllPresentLinks(removeEmployment: By, removeEmployer: By): Unit = {
```

Comment:
This method is too specific for a Base class. The behavior should be added to the step definition implementation.